tests/rocprof-sys-testing.cmake

```
@@ -376,6 +376,54 @@ ${_FILE_CONTENTS}
         PARENT_SCOPE)
 endfunction()
 
+# -------------------------------------------------------------------------------------- #
+# Take input as regex and return match from rocminfo
+function(rocprofiler_sys_get_gfx_info _REGEXP _VAR)
+    cmake_parse_arguments(ARG "ECHO" "PREFIX;DELIM" "" ${ARGN})
```

```suggestion
function(rocprofiler_sys_get_gfx_info _VAR)
    cmake_parse_arguments(ARG "ECHO" "PREFIX;DELIM;GFX_MATCH" "" ${ARGN})
```
Updated argument parsing

---

Taken changes